assert
.
throws
(
TypeError
function
(
)
{
Function
.
prototype
.
call
.
call
(
undefined
{
}
)
;
}
'
Function
.
prototype
.
call
.
call
(
undefined
{
}
)
throws
a
TypeError
exception
'
)
;
reportCompare
(
0
0
)
;
